lib/iris/tests/unit/fileformats/grib/message/test_GribMessage.py

```diff
@@ -91,46 +91,21 @@ def test_no_bitmap(self):
         self.assertEqual(result.shape, self.shape)
         self.assertArrayEqual(result, expected)
 
-    @tests.skip_dask_mask
-    def test_bitmap_present_int_data(self):
+    def test_bitmap_present(self):
```

Comment:
I'm missing something. Why don't we need to test for floats as well as integers?